bigquery/tests/unit/test_job.py

```diff
@@ -2720,6 +2720,70 @@ def test_reload_w_alternate_client(self):
         self.assertEqual(req['path'], PATH)
         self._verifyResourceProperties(job, RESOURCE)
 
+    def test_to_dataframe(self):
```

Comment:
Again, use `unittest.skipUnless(_HAVE_PANDAS)`.

Reply:
Added skips in 70d6bd9